src/components/Chart/Chart.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import C3Chart from 'react-c3js';
+import ClassNames from 'classnames';
+
+require('patternfly/dist/js/patternfly-settings');
+require('patternfly/dist/js/patternfly-settings-charts');
+
+const c3ChartDefaults = window.patternfly.c3ChartDefaults();
+
+/**
+ * Chart Component for Patternfly React
+ */
+const Chart = ({ ...props }) => {
+  const chartType = props.data.type;
```

Comment:
`const chartType = props.data && props.data.type; ` just in case?

Reply:
if `props.data` is undefined we get a lot of errors ... no just here ... it should be required ...